psydac/pyccel/ast/datatypes.py

```diff
@@ -75,9 +75,11 @@
                                  'int32':('int',4),
                                  'int64':('int',8),
                                  'int'  :('int', default_precision['int']),
+                                 'int_'  :('int', default_precision['int']),
```

Comment:
This is for `numba` , it only accepts `bool_` and `int_` as dtypes and not `bool` or `int`, because `bool` and `int` are Python datatypes, [here](https://stackoverflow.com/questions/55905690/how-exactly-does-the-behavior-of-python-bool-and-numpy-bool-differ) they explain the difference in more details.

Reply:
no we don't , we probably need to

Reply:
Please add a comment next to the new datatypes. They will be very useful to the developers who are going to remove the `pyccel` subdirectory from Psydac!

Reply:
done